src/FileConventions.Test/FileConventions.Test.fsproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
     <PackageReference Include="NUnit.Analyzers" Version="3.3.0" />
     <PackageReference Include="coverlet.collector" Version="3.1.2" />
-    <PackageReference Include="Fsdk" Version="0.6.0--date20230214-0422.git-1ea6f62" />
+		<PackageReference Include="Fsdk" Version="0.6.0--date20230821-0702.git-5488853" />

```

Comment:
@Mersho hey Mershad, do you see the issue here ^ ? this means you have your IDE misconfigured please fix it; and also means that you're not looking at your own diffs (to detect this kind of things before you commit; or looking at the diff of your own commit or PR in github after you have pushed); are you familiar with the tool `git gui`?